upgrade/rendering-changes/list.md

```diff
@@ -0,0 +1,135 @@
+---
+title: List of Versions
+description: See which versions have rendering changes so you can take them into account when upgrading.
+page_title: List of Versions with Rendering Changes
+slug: versions-with-rendering-changes
+position: 0
+---
+
+# List of Telerik UI for Blazor Versions with Rendering Changes
+
+
+As the Telerik UI for Blazor matures we will continuously optimize the rendering and the usage of CSS classes. The rendering changes will affect the styling of the components if the business application overrode the built-in CSS rules. This article will track the version where such changes are introduced.
```

Comment:
Names are not used with articles, unless they play the role of an adjective:
* incorrect - "the Telerik UI for Blazor matures"
* correct - "the Telerik UI for Blazor components mature"

```suggestion
As Telerik UI for Blazor matures, we may optimize the HTML rendering and the usage of CSS classes. The rendering changes will affect the styling of the components if the application uses custom CSS that overrides the built-in styles.

This article will track the versions where such changes were introduced.
```